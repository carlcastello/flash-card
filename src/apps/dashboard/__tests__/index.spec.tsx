import React from 'react';
import Web  from '../index';
import { shallow } from 'enzyme';

describe("Web APP", () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = shallow(<Web/>);
  });


  it("Takes snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  })
});