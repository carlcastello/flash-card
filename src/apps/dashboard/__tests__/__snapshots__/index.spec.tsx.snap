// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Web APP Takes snapshot 1`] = `ShallowWrapper {}`;
